t/compile.t

```diff
@@ -18,11 +18,8 @@ map {
     }, catfile($thisdir, $_));
 } qw(../bin ../lib);
 
-for (@tests) {
-    my $output = `perl -I "$libpath" -c $_ 2>&1`;
-    my $ok = ($? >> 8 == 0);
-    print $output unless $ok;
-    ok($ok, "$_ syntax check");
+foreach my $file (@tests) {
+    require_ok $file;
```

Comment:
> What do you think about just switching to Test::Compile? Seems like the easiest thing to do

Well, the easiest thing to do is is to do nothing, as what we already have now is a few lines of code that check for compilation errors and don't require any external dependency. Bringing in external dependencies should also bring some obvious benefits.